internal/gitutil/git.go

```diff
@@ -78,6 +79,9 @@ func GetRemoteURL(dir string) (string, error) {
 func Exec(pwd string, args ...string) (string, error) {
 	klog.V(4).Infof("Going to run git %s", strings.Join(args, " "))
 	cmd := osexec.Command("git", args...)
+	if runtime.GOOS == "windows" {
+		cmd.Env = append(os.Environ(), "MSYS=noglob")
```

Comment:
If you prefer to append CYGWI=noglob also, please let me known.

Or we should append the variable like below?
```
msys := os.Getenv("MSYS") + ":noglob"
```